frontend/src/app/auth.service.ts

```diff
@@ -22,6 +23,12 @@ export class AuthService {
   private loginEvent = new Subject<SuccessOrFailureAction>();
   private resetPasswordEvent = new Subject<SuccessOrFailureAction>();
   private confirmResetPasswordEvent = new Subject<SuccessOrFailureAction>();
+  private _loginUrl = `${environment.apiRoot}/login`;
```

Comment:
it looks like some merge went pretty awfull - this urls were moved stripped from `_` prefix and are available above